ui/app/clinical/common/services/cdssService.js

```diff
@@ -0,0 +1,229 @@
+'use strict';
+
+angular.module('bahmni.clinical')
+    .service('cdssService', ['drugService', function (drugService) {
```

Comment:
In general there is a lot of "code" that is written for CDSS functionality to work. I think maybe we can add a check for whether cdss functionality is enabled / not first.. and if its not enabled, we don't need to register any of the methods/listeners --- to keep page performance / events/etc at a minimal. What do you think? 

//At the start of the code. 
if (cdss == false) return;